app.py

```diff
@@ -111,14 +111,14 @@ def graph_refresh_solver() -> None:
         _THOTH_GRAPH_REFRESH_EAGER_STOP,
     )
 
-    indexes = list(_GRAPH_DB.get_python_package_index_urls())
```

Comment:
`list` is not required anymore, as `get_python_package_index_urls_all` gives back a `List[str]` already.

Reply:
Thanks!